### PR TITLE
kirigami-addons: New package

### PR DESCRIPTION
**Pull request by BLumia**
Resolve https://github.com/msys2/MINGW-packages/issues/13799

BTW, should I replace the "Maintainer" comment header to myself? If the answer is yes, then what's the policy to update the package? Do I need to follow any extra setup (e.g. email to notify outdated?) to maintain this package?

### CODE REVIEW

**Comment by Biswa96** [comment]
You can replace the maintainer tag to yourself. There is no hardcore policy.

File: mingw-w64-kirigami-addons-qt5/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  _kde_f5_add_depends "${_variant}" "${MINGW_PACKAGE_PREFIX}-qt5${_namesuff}"
+else
+  makedepends+=("${MINGW_PACKAGE_PREFIX}-qt5-tools")
+  depends+=("${MINGW_PACKAGE_PREFIX}-qt5-base"
+            "${MINGW_PACKAGE_PREFIX}-qt5-declarative"
+            "${MINGW_PACKAGE_PREFIX}-qt5-quickcontrols2"
+            "${MINGW_PACKAGE_PREFIX}-qt5-svg")
+fi
+groups=("${MINGW_PACKAGE_PREFIX}-kf5")
+source=("https://download.kde.org/stable//${_basename}/${_basename}-${pkgver}.tar.xz"{,.sig})
```
Comment (Biswa96) [inline mingw-w64-kirigami-addons-qt5/PKGBUILD:28]:
```suggestion
source=("https://download.kde.org/stable/${_basename}/${_basename}-${pkgver}.tar.xz"{,.sig})
```

Comment (BLumia) [reply mingw-w64-kirigami-addons-qt5/PKGBUILD]: Patch updated :)

**Comment by Biswa96** [comment]
@lazka Is it possible to save and use the PGP keys in the package folder like ArchLinux does?

**Comment by lazka** [comment]
Sounds good to me. I don't know how that works though (does makepkg handle it? or do we need to expand https://github.com/msys2/MINGW-packages/blob/master/.ci/fetch-validpgpkeys.sh to import files from a local directory?)

**Comment by BLumia** [comment]
I'm actually taking [kitemmodels' PKGBUILD](https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-kitemmodels-qt5/PKGBUILD) as a reference and see that file is trying to validate the PGP key. Not sure why that one is successed but this one failed. Nether kitemmodels' nor this one can be validate on my machine, tho.

Should I remove the PGP validation part from this patch?

**Comment by Biswa96** [comment]
> Should I remove the PGP validation part from this patch?

Yes, please. You can just remove the `.sig` part from the `source` variable and then `updpkgsums`.

**Comment by BLumia** [comment]
Patch updated :)

**Comment by Biswa96** [comment]
kirigami-addons needs kirigami2, see https://github.com/archlinux/svntogit-packages/blob/packages/kirigami-addons/trunk/PKGBUILD

**Comment by BLumia** [comment]
Sorry I should pay more attention to CI result before :joy: Should be fixed now.

btw, is there a way to test if a PKGBUILD is missing required `depends` locally without uninstalling the packages in my current working environment?

**Comment by Biswa96** [comment]
As far as I know, `ntldd` can show the dependent DLLs but one has to figure out the dependencies manually. `ntldd` output is somewhat verbose.

**Pull request merged**